<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>交互式数字蒙太奇</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="compositeImage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>560</width>
      <height>320</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>359</y>
      <width>760</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>754</width>
       <height>195</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="remove"/>
    <addaction name="removeAll"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helper"/>
    <addaction name="aboutUs"/>
   </widget>
   <widget class="QMenu" name="mode">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="mode0"/>
    <addaction name="mode1"/>
    <addaction name="mode2"/>
    <addaction name="mode3"/>
    <addaction name="mode4"/>
    <addaction name="mode5"/>
    <addaction name="mode6"/>
    <addaction name="mode7"/>
    <addaction name="mode8"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="generate"/>
    <addaction name="viewResult"/>
    <addaction name="saveResult"/>
   </widget>
   <widget class="QMenu" name="adjParam">
    <property name="title">
     <string>调参</string>
    </property>
    <addaction name="inputParam"/>
   </widget>
   <addaction name="file"/>
   <addaction name="menu"/>
   <addaction name="mode"/>
   <addaction name="adjParam"/>
   <addaction name="help"/>
  </widget>
  <action name="openFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导入图片</string>
   </property>
  </action>
  <action name="generate">
   <property name="text">
    <string>生成图像</string>
   </property>
  </action>
  <action name="helper">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="aboutUs">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="remove">
   <property name="text">
    <string>移除图片</string>
   </property>
  </action>
  <action name="removeAll">
   <property name="text">
    <string>移除全部</string>
   </property>
  </action>
  <action name="mode0">
   <property name="text">
    <string>USER_SPECIFY</string>
   </property>
  </action>
  <action name="mode1">
   <property name="text">
    <string>MAX_LUMIN</string>
   </property>
  </action>
  <action name="mode2">
   <property name="text">
    <string>MIN_LUMIN</string>
   </property>
  </action>
  <action name="mode3">
   <property name="text">
    <string>ERASE</string>
   </property>
  </action>
  <action name="mode4">
   <property name="text">
    <string>MAX_LIKEHOOD</string>
   </property>
  </action>
  <action name="mode5">
   <property name="text">
    <string>MIN_LIKEHOOD</string>
   </property>
  </action>
  <action name="mode6">
   <property name="text">
    <string>CONTRAST</string>
   </property>
  </action>
  <action name="mode7">
   <property name="text">
    <string>MAX_DIFF</string>
   </property>
  </action>
  <action name="mode8">
   <property name="text">
    <string>USER_SPECIFY_P</string>
   </property>
  </action>
  <action name="changeBase">
   <property name="text">
    <string>更换底图</string>
   </property>
  </action>
  <action name="viewResult">
   <property name="text">
    <string>查看结果</string>
   </property>
  </action>
  <action name="saveResult">
   <property name="text">
    <string>保存结果</string>
   </property>
  </action>
  <action name="inputParam">
   <property name="text">
    <string>输入参数</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
